# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/mac/moz.build
# Commit: 5194f8d4d589
# Full Hash: 5194f8d4d5890aafe62c42f9b1fa75d6a9a64bf5
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P12. Allow ffvpx and the AppleDecoderModule in the RDD. r=haik,jolin
#   
#   Ultimately, we should be able to remove everything that got added to the RDD sandbox from the content's one.
#   
#   Fly-by fix; allow checking if AVX512 is supported in content sandbox.
# ==============================================================================

diff -r 84b82a60dc74 -r 5194f8d4d589 security/sandbox/mac/moz.build
--- a/security/sandbox/mac/moz.build	Tue Oct 20 23:27:36 2020 +0000
+++ b/security/sandbox/mac/moz.build	Tue Oct 20 23:27:52 2020 +0000
@@ -9,6 +9,7 @@
     'SandboxPolicyContent.h',
     'SandboxPolicyFlash.h',
     'SandboxPolicyGMP.h',
+    'SandboxPolicyRDD.h',
     'SandboxPolicyUtility.h'
 ]
 
